learning_journal/tests/test_models.py

```diff
@@ -10,3 +10,19 @@ def test_create_entry(dbtransaction):
     DBSession.add(new_model)
     DBSession.flush()
     assert new_model.id is not None
+
+
+def test_created(dbtransaction, new_entry):
+    assert new_entry.created is not None
+
+
+def test_id(dbtransaction, new_entry):
+    assert new_entry.id is not None
+
+
+def test_post_title(dbtransaction, new_entry):
+    assert new_entry.title is not None
+
+
+def test_post_text(dbtransaction, new_entry):
+    assert new_entry.text is not None
```

Comment:
What about tests that prove the constraints on your model.  Prove that you can't make one that has a title longer than *n* characters.